namespaces/kubernetes.yml

```diff
@@ -94,6 +94,131 @@ namespace:
         description: >
           The provider associated with this Kubernetes object
 
+  - name: event
+    type: group
+    description: >
+      The kubernetes event obtained from kubernetes master API
+      The event is already JSON object and as whole nested under kubernetes field
+      This description should loosely follow 'type Event' in https://github.com/kubernetes/kubernetes/blob/master/pkg/api/types.go
+    fields:
+      - name: verb
+        type: string
+        example: ADDED
+        description: >
+          The type of event, can be ADDED, MODIFIED, DELETED
+      - name: metadata
+        type: group
+        description: >
+          Information related to the location and time of the event creation
+        fields:
+          - name: name
+            type: string
+            example: java-mainclass-1.14d888a4cfc24890
+            description: >
+              Name of the entity that induced the event creation
+          - name: namespace
```

Comment:
my guess is event of the namespace as a resource vs the namespace of the object the event is about. But it'll be interesting to learn which one is "better". Eventually the event will need to be routed to index based on namespace.